tests/utils/test_temp_env_on_cpu.py

```diff
@@ -0,0 +1,145 @@
+# Copyright 2024 Bytedance Ltd. and/or its affiliates
```

Comment:
I guess we have to use pytest, otherwise it won't be invoked.

Reply:
pytest is covering all test_xx files https://github.com/volcengine/verl/blob/main/.github/workflows/cpu_unit_tests.yml#L83